docs/docs/welcome.mdx

```diff
@@ -3,6 +3,8 @@ slug: /
 title: Welcome to Nebari's documentation
 ---
 
+<!-- TODO: Remove in favour of the new website gh: #182-->
```

Comment:
The content in this doc, though, doesn't seem equal to the landing page content.

Reply:
It is not and should not be the same content in the landing page. 
This doc is massively outdated. The landing page content should be in sec/pages/index.mdx